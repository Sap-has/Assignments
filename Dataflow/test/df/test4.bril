@main {
  i: int = const 0;
  sum: int = const 0;
.loop:
  max: int = const 5;
  cond: bool = lt i max;
  br cond .body .end;
.body:
  sum: int = add sum i;
  one: int = const 1;
  i: int = add i one;
  jmp .loop;
.end:
  print sum;
}