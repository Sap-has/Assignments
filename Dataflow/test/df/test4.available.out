b1:
  in:  ∅
  out: ∅
loop:
  in:  ∅
  out: lt i max
body:
  in:  lt i max
  out: add i one, add sum i
end:
  in:  lt i max
  out: lt i max
